### PR TITLE
Fix #5223 - Menu actions translation for develop branch

### LINKED ISSUE #5223: Menu ACTIONS its not translatable - SuiteP - version 7.10.RC2
#### Issue

The menu "ACTIONS" in SuiteP template is not translatable.
7.9.x works fine but the fix from https://github.com/salesagility/SuiteCRM/issues/1915 is partially not implemented in 7.10

2018-02-15 15_52_13-inicio

#### Possible Fix

themes/SuiteP/include/MySugar/tpls/MySugar.tpl 
needs to be changed to
`<a class="dropdown-toggle" data-toggle="dropdown" href="#">{$dashboardActions}</a>`

I will submit a PR

#### Your Environment

* SuiteCRM Version used: 7.10.RC2
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Chrome
* Environment name and version (e.g. MySQL, PHP 7): apache 2.4.18 / PHP Version 7.0.11 / mysql 5.7.17
* Operating System and version (e.g Ubuntu 16.04):  Windows 10 - 32 bit - local Wamp

### PR DESCRIPTION
**Pull request by horus68**
## Description

Menu Actions for accounts / profiles or cases can't be translated. This PR fixes bug reported here #5223
Note: hotfix branch is ok, only developer branch and 7.10.rc2 have the problem after this other edit:
https://github.com/salesagility/SuiteCRM/commit/2ed1f235a50caff15e94e8776b0e182865221289#diff-f83ca3eb1c411186a99fc09dca107ed1

Accounts view Before Fix (Portuguese language):
2018-02-15 15_52_13-inicio

After fix (repair and rebuild is recommended):
2018-02-15 17_00_21-inicio

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

**Comment by horus68** [comment]
@Dillon-Brown As the hotfix branch is ok (the fix has is almost 2 years), only developer branch and 7.10.RC2 full zip are wrong.
So from what branch is created the RC2 versions?

**Comment by horus68** [comment]
The problem was created here:
https://github.com/salesagility/SuiteCRM/commit/2ed1f235a50caff15e94e8776b0e182865221289#diff-f83ca3eb1c411186a99fc09dca107ed1

**Comment by horus68** [comment]
Update: new error with this fix the main screen Actions menu shows the word duplicated.
The other screens are fine!
So there must be another place to fix
2018-02-15 17_18_22-inicio

File: themes/SuiteP/include/MySugar/tpls/MySugar.tpl [hunk 1]
```diff
@@ -98,7 +98,7 @@
 
         {if !$lock_homepage}
             <li id="tab-actions" class="dropdown">
-                <a class="dropdown-toggle" data-toggle="dropdown" href="#">{$dashboardActions}</a>
+                <a class="dropdown-toggle" data-toggle="dropdown" href="#">{$dashboardActions}{$APP.LBL_LINK_ACTIONS}</a>
                 {include file='themes/SuiteP/include/MySugar/tpls/actions_menu.tpl'}
```
Comment (PedroErnst) [inline themes/SuiteP/include/MySugar/tpls/MySugar.tpl]: Should {$dashboardActions} maybe be removed/replaced by {$APP.LBL_LINK_ACTIONS}?

Comment (horus68) [reply themes/SuiteP/include/MySugar/tpls/MySugar.tpl]: @PedroErnst after that change it worked fine!

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**